m2cgen/interpreters/r/interpreter.py

```diff
@@ -18,6 +17,8 @@ class RInterpreter(ToCodeInterpreter,
     # This value is just a heuristic and is subject to change in the future
     # based on the users' feedback.
     bin_depth_threshold = 25
+    ast_size_check_frequency = 2
```

Comment:
Depending on a Travis build additional tuning may be required. The execution time looks reasonable when running locally.